mlflow/tracing/provider.py

```diff
@@ -345,6 +345,11 @@ def _get_span_processors(disabled: bool = False) -> list[SpanProcessor]:
     #  3. MLflow will pick the implementation based on given destination id.
     trace_destination = _MLFLOW_TRACE_USER_DESTINATION.get()
     if trace_destination and isinstance(trace_destination, (MlflowExperiment, Databricks)):
+        if is_in_databricks_model_serving_environment():
+            _logger.warning(
```

Comment:
After a second thought, maybe we should use INFO 🤔 
```suggestion
            _logger.info(
```